web/src/pages/Register/signup.scss

```diff
@@ -0,0 +1,203 @@
+
```

Comment:
You should remove this, this directory is already pushed in current under a new name